_sass/1-modules/_colors.scss

```
@@ -0,0 +1,26 @@
+// Colors =======================================================
+:root {
+  --text: #222;
+  --text_subtle: #999;
+  --subtle: #ddd;
+  --subtle_: #aaa;
+  --link: blue;
+  --link_subtle: #acb9ce;
+  --bg: #fff;
+  --bg_: #f5f5f5;
+  --bg__: #fff;
+  --transparent: #ffffff00;
+
+  @media (prefers-color-scheme: dark) {
+    --text: #ddd;
+    --text_subtle: #666;
+    --subtle: #353535;
+    --subtle_: #555;
+    --link: #3C81F9;
+    --link_subtle: #435472;
+    --bg: #111;
+    --bg_: #181818;
+    --bg__: #222222;
+    --transparent: #00000000;
```

Use an actual transparent color instead of hex hack? Up to you!

```suggestion
    --transparent: rgba(0,0,0,0);
```

---

actually, how do you feel about hsl colors in general? would you be opposed to this?

---

I mean, hex transparent is technically valid. I just didn't want to introduce many color types if hex was the main way we communicated 😄